webbpsf/__init__.py

```diff
@@ -13,119 +13,42 @@
 Documentation can be found online at https://webbpsf.readthedocs.io/
 """
 
-import sys
-from astropy import config as _config
-
-try:
-    from .version import __version__
-except ImportError:
-    __version__ = ""
-
-__minimum_python_version__ = "3.10"
-
 
-class UnsupportedPythonError(Exception):
-    pass
+import os
+import stpsf
+import sys
+import warnings
 
 
-if sys.version_info < tuple(
-    (int(val) for val in __minimum_python_version__.split("."))
-):
-    raise UnsupportedPythonError(
-        "webbpsf does not support Python < {}".format(__minimum_python_version__)
-    )
+_warned = False
 
+if not _warned:
+    warnings.warn(
```

Comment:
As discussed in slack, I'm not sure a multi-line warning with lots of `!!!!!` is the best choice at this time.  Suggest making it shorter, and making a webpage in the docs for more information on the name migration. 

Perhaps like: 
```
"""
This message is for information only and WebbPSF will continue to function as normal.
The WebbPSF library has been moved/renamed to STPSF. Please see [READTHEDOCS_PAGE_TBD] for more information. 
WebbPSF is now an alias of STPSF and is running code from the STPSF library.
"""

```

And then that webpage, some new child page within the stpsf docs, can give more details and example code, rather than trying to pack too much into a warning message.

Reply:
Replacing exactly as suggested here with a TODO until we have a proper page with instructions set up (stpsf 2.0.1)